### ISSUE TITLE
[OS X] High contrast theme: weird blue line on top

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#3275]
#3248 
Notice the weird blue line on top. What does it represent - since it is not focus

![screen shot 2016-02-23 at 10 50 39](https://cloud.githubusercontent.com/assets/1926584/13247738/5090d0c6-da1b-11e5-97e1-e1a7b64293e9.png)

**Comment by bgashler1** [comment]
This comes from the `global-message-list` div.  It should be hidden when not being used.  However it's visible here even when there's no message.
```
<div class="global-message-list" aria-hidden="false" style="top: 0px;"><ul class="message-list"></ul></div>
```

Adding the class `hidden` should fix the problem.  @isidorn would you be able to handle this one since it's more logic-oriented?

**Comment by isidorn** [comment]
Sure, I can look into it.

**Comment by isidorn** [comment]
Somebody fixed this and it was not me

**Comment by bgashler1** [comment]
I'm still seeing this issue.

**Comment by isidorn** [comment]
@bgashler1 please provide more details, do you see it by just switching to high contrast theme. What version of vscode are you using? Do you see it if you run latest out of dev?

**Comment by bgashler1** [comment]
I see it on my up-to-date local build and on Alpha.  It seems to only happen whenever there was an alert message and it is dismissed (the blue bar remains after it is dismissed).

This blue bar is the border-outline from the collapsed `<ul class="message-list"></ul>`.

I just fixed this with a simple CSS rule.  I'll be pushing momentarily

**Comment by isidorn** [comment]
Awesome!
